admin-dev/themes/new-theme/js/components/entity-search-input.ts

```diff
@@ -418,4 +447,20 @@ export default class EntitySearchInput {
 
     return template;
   }
+
+  private getSelectedIds(): any[] {
```

Comment:
An input value is always a string, I think you can replace this and types above with `Array<string>` or `string[]`

Reply:
Even for a `<input type="number" />` ?